src/pages/Proposal/index.tsx

```diff
@@ -53,6 +57,19 @@ export const ProposalUi: React.FC<ProposalUiProps> = ({
     <div css={styles.root}>
       <ProposalSummary css={styles.summary} proposal={proposal} />
 
+      {!voteProposalFeatureEnabled && (
+        <div className="mb-8 w-full" data-testid={TEST_IDS.votingDisabledWarning}>
+          <NoticeInfo
+            title={t('vote.multichain.votingOnlyEnabledOnBsc')}
+            description={
+              <Button variant="text" onClick={() => switchChain({ chainId: chains[0].id })}>
```

Comment:
According to the designs, the button has no set height or vertical padding:
```suggestion
              <Button className="h-auto" variant="text" onClick={() => switchChain({ chainId: chains[0].id })}>
```